docs/content/configuration/creating_volumesnapshotclass.md

```diff
@@ -11,12 +11,13 @@ When configuring the file, be sure to use the same array secret and array secret
 -   The `snapshot_name_prefix` parameter is optional.
 
     **Note:** For IBM DS8000® family, the maximum prefix length is five characters.<br/>The maximum prefix length for other systems is 20 characters.<br/>For storage systems that use Spectrum Virtualize, the `CSI` prefix is added as default if not specified by the user.
+
+- The `virt_snap_func` parameter is optional but necessary in Spectrum Virtualize storage systems if using the Snapshot function. To enable the Snapshot function, set the value to _true_. The default value is _false_. If the value is `false` the snapshot will use the FlashCopy function.
```

Comment:
is it clear that the value (true/false) should be a string (in "" like the example)? `true` and `"true"` are different values (the first is type boolean and the other is type string and we want string)

Reply:
per our discussion all input needs to have the quote (") before and after the word. Updated. please verify and resolve

Reply:
looks good